lang/ruby/test/test_protocol.rb

```diff
@@ -196,4 +215,9 @@ def test_namespaces
       assert_equal type.namespace, 'com.acme'
     end
   end
+
+  def test_support_for_doc_attribute
+    original = Protocol.parse(EXAMPLES.last.protocol_string)
+    assert_equal original.types.first.fields.first.doc, 'documentation'
```

Comment:
This isn't testing the doc attribute on a Message, which is added implemented in this PR, just doc on a Field.